<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-EBL30HHN" failures="1" tests="1" name="Default test" time="0.732" errors="0" timestamp="2024-09-03T00:34:13 IST">
  <testcase classname="RAW.Tests.TestHomePage" name="Test1" time="0.701">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
[public void RAW.Tests.TestHomePage.Test1(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: Test1([java.lang.String arg0, java.lang.String arg1])
Arguments: [(org.apache.poi.xssf.usermodel.XSSFCell) anu@example.com,(org.apache.poi.xssf.usermodel.XSSFCell) Anu@1234]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void RAW.Tests.TestHomePage.Test1(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: Test1([java.lang.String arg0, java.lang.String arg1])
Arguments: [(org.apache.poi.xssf.usermodel.XSSFCell) anu@example.com,(org.apache.poi.xssf.usermodel.XSSFCell) Anu@1234]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:918)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="RAW.Tests.TestHomePage" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- Default test -->
